packages/wonder-blocks-clickable/src/util/__tests__/is-external-url.js.test.js

```diff
@@ -1,28 +1,33 @@
 // @flow
-import isExternalUrl from "../is-external-url.js";
+import {isClientSideUrl} from "../is-client-side-url.js";
```

Comment:
What do you think about changing this file name from `is-external-url.js.test.js` to `is-client-side-url.js.test.js` to match the file that is being tested?